docusaurus/docs/engineering/start-developing/boba-backend.md

```diff
@@ -10,19 +10,36 @@ sidebar_position: 4
 
 ## Install Instructions
 
-The first time you work on BobaBackend you need to copy the project to the local machine and set up the environment configuration files (`.env`, [ELI5](https://www.reddit.com/r/webdev/comments/a54pxr/what_is_a_env_file/ebjwbtr/?utm_source=reddit&utm_medium=web2x&context=3)).
+The first time you work on BobaBackend you'll need to [fork the repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo#forking-a-repository) so you can edit your own version of the code.
 
-### 1 – Clone project and create configuration files
+### 1 — Clone your fork
 
-In a terminal, run the following commands:
+In a terminal, run the following commands - or [follow Github's instructions](https://docs.github.com/en/get-started/quickstart/fork-a-repo#cloning-your-forked-repository).
 
```

Comment:
should we... mention that you should cd into the folder where you store your projects before you clone?